src/main/ui/app/route/TagPage.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import {Component} from "react";
+import fetch from "isomorphic-fetch";
+import AddOnList from "../component/AddOnList";
+
+export default class TagPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    componentDidMount() {
+        this.getTag();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.location.query !== prevProps.location.query) {
+            this.getTag();
+        }
+    }
+
+    getTag() {
+        const tag = this.props.location.query.tag;
+    
+        console.log(tag);
```

Comment:
Sorry, in my head this was implied by one of my other comments, but I should have been explicit!

To correctly implement this feature, we should support `https://addons-stg.openmrs.org/#/search?&tag=form-entry`

Basically we should change the underlying search functionality so that it can accept 3 different optional parameters: q, type, tag.

The REST endpoint should also accept all three of these, and pass them down to the Java API.

And I assume that would allow us to use the same search page to handle everything without duplicating.

(But if that doesn't work for some reason it's okay to do what you're doing here; I haven't closely reviewed this file, but I will do so. I assume it's a lot of copy-paste from the List page?)